#goal definition
float32 distance
---
#result definition
float32 distanceTraversed
---
#feedback
float32 distanceTotal
